metron-platform/metron-indexing/src/test/java/org/apache/metron/indexing/dao/UpdateIntegrationTest.java

```diff
@@ -106,14 +106,14 @@ public void test() throws Exception {
       }}, Optional.empty());
 
       Assert.assertEquals(message0, update.getDocument());
-      Assert.assertEquals(1, getMockHTable().size());
+      Assert.assertEquals(10, getMockHTable().size());
```

Comment:
Since I am reusing the `IndexDao` to load the tests data, the HBase table also gets this test data now.